experimenter/experimenter/segments/tests/fixtures/valid_segments_definitions/firefox_desktop.toml

```diff
@@ -0,0 +1,14 @@
+[segments.desktop_segment_1]
+friendly_name = "Desktop Segment 1"
+description = "Firefox desktop segment used for testing"
+select_expression="country_code in ('IN', 'US')"
+
+[segments.desktop_segment_2]
+friendly_name = "Desktop Segment 2"
+description = "Firefox desktop segment used for testing"
+select_expression="country_code in ('IN', 'US')"
+
+[segments.desktop_segment_3]
+friendly_name = "Desktop Segment 3"
+description = "Firefox desktop segment used for testing"
+select_expression="country_code in ('IN', 'US')"
```

Comment:
It might be more valuable to configure these slightly differently so that the test covers more cases. For example, if some fields aren't required, you can test that by leaving some out in segment 2 and others in segment 3 (or add a couple new ones) and make sure the tests still pass. It looks like you'll need to tweak the tests to handle this change too.